dts/bindings/espi/nuvoton,npcx-espi.yaml

```diff
@@ -30,3 +30,13 @@ properties:
 
         For example the WUI mapping on NPCX7 would be
            espi-rst-wui = <&wui_cr_sin1>;
+
+  rx-plsize:
+    type: int
+    required: true
+    description: The amount of flash receive buffer registers.
```

Comment:
this sounds a bit odd :-) maybe "The maximum receive channel payload size"? (the define comment is "Flash channel maximum payload size")

Reply:
Done.